### ISSUE TITLE
Chl-a derived from Flu calculated using transfer function

### ISSUE THREAD

**Issue by ggalibert** [aodn/imos-toolbox#544]
At the last QC summit @NickMortimer showed that Chl-a derived from Fluorescence should be calculated using a transfer function with calibration information. This operation could be performed by the toolbox.

1. @NickMortimer could you please provide some documentation on how to apply this transfer function?
2. How do we make calibration information available to the toolbox?

**Comment by ggalibert** [comment]
Initial draft document on this [Developing a protocol for calibrating field fluorometers_Doubell_Draftv2.._.pdf](https://github.com/aodn/imos-toolbox/files/3146145/Developing.a.protocol.for.calibrating.field.fluorometers_Doubell_Draftv2._.pdf) from Mark Doubell.

Please @pvanruth, @anaredd, @charlesjames-jan2017 feel free to comment/respond to the questions above.

**Comment by ocehugo** [comment]
We also need to provide a policy on how to provide this calibrated measurement.

my thinking:
1. Provide the uncalibrated data.
2. Provide the calibrated data.
3. Provide the coefficient for calibration in the calibrated variable metadata.

**Comment by NickMortimer** [comment]
It is my understanding that this function can be set in the instrument and as such included in the data file. for WQMs and the JFE there are two calibrations available user and factory. At the moment the tool box does not record these.

**Comment by doobsmj** [comment]
Hi, I'm now included in GitHub. :) 

In response to the first post by ggalibert on 2-May.

We are currently costing up the approach to conduct a calibration trail using the methods outlined in the protocol (pdf) document.
Answers to the questions:
1. This will be an outcome of the calibration procedure and will be a simple linear function for each instrument 
2. I suggest the the people/facility  performing the calibration will need to 1) provide to the toolbox managers to enter into the toolbox or 2) or provide to instrument owners and for the toolbox to allow a way for users to enter the provided calibration coefficients

**Comment by ocehugo** [comment]
@doobsmj  welcome mate :)

After reading the draft above, I'm thinking that having the actual data that (both) lab types of equipment produce are essential. This should go, as auxiliary data/metadata,  into the actual netCDF file with the deployed fluorescence equipment data. The reason why is that anyone can fit any model to it and/or verify them. For example, the default will be committee defined one  (say a linear regression), but the data will be there for anyone to try to evaluate/improve/propose new things/etc.

I think it's important to have this information within the file, at least in FV00 ones.

The data also needs to be within a proper and accessible database, so IMOS and/or the calibration group can do nice planning stuff - estimate calibration dates, costs, and other management things.

**Comment by NickMortimer** [comment]
I agree that raw counts should be in the data along with a lab calibrated value. then if possible a value that is adjusted using the water bottles.

all the coefficients for the transformations should be in the data file
